src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/single-internal-link.scss

```diff
@@ -0,0 +1,54 @@
+.single-internal-link {
+    cursor: pointer;
+
+    &.container {
+        position: relative;
+
+        width: 100%;
+        height: $inputHeight + 2; // border height
+        border: 1px solid $grayLighterer;
+        padding: 0 5px;
+        font-size: $baseFontSize;
+        border-radius: $borderRadius;
+        background: white;
```

Comment:
Would use the `$white` variable, in case we replace it one day for something little bit less white :wink: